lib/actions/auth.coffee

```diff
@@ -88,6 +88,11 @@ exports.login	=
 						name: 'token'
 						type: 'input'
 				.then(resin.auth.loginWithToken)
+				.tap ->
+					resin.auth.whoami()
+					.then (username) ->
+						if !username
+							throw new resin.errors.ResinMalformedToken(options.token)
```

Comment:
I'm not sure I understand. Why is it misleading?

Reply:
I would expect an "invalid credentials" or "unathorized" error instead of "malformed".
I guess that it's because I was using an api key for the wrong environment, otherwise this would probably make sense to normal users.
Let's just ignore this.

Reply:
No, I think you're totally right.

We don't actually need to limit ourselves to SDK errors at all here, so I've updated this to throw an expected error (which means it doesn't go to Sentry) with much clearer text. Failure now looks like:

```
resin login --token asdasd                          
______          _         _
| ___ \        (_)       (_)
| |_/ /___  ___ _ _ __    _  ___
|    // _ \/ __| | '_ \  | |/ _ \
| |\ \  __/\__ \ | | | |_| | (_) |
\_| \_\___||___/_|_| |_(_)_|\___/

Logging in to resin.io
Token authentication failed

If you need help, don't hesitate in contacting us at:

  GitHub: https://github.com/resin-io/resin-cli/issues/new
  Forums: https://forums.resin.io
```

Better?